sdk/core/System.ClientModel/src/Internal/StreamExtensions.cs

```diff
@@ -86,4 +89,45 @@ public static async Task WriteAsync(this Stream stream, ReadOnlySequence<byte> b
                 ArrayPool<byte>.Shared.Return(array);
         }
     }
+
+    public static async Task CopyToAsync(this Stream source, Stream destination, CancellationToken cancellationToken)
```

Comment:
why do we need this method? Stream already has a method to copy from one stream to another.

Reply:
It is not available on netstandard2.0: https://learn.microsoft.com/en-us/dotnet/api/system.io.stream.copytoasync?view=net-8.0#system-io-stream-copytoasync(system-io-stream-system-threading-cancellationtoken)